### PR TITLE
GH-1796 - Highlight Gnosis Safe in whats new

### PR DESCRIPTION
**Pull request by biafra23**
Handles #1796 

Changes proposed in this pull request:
image

@gnosis/mobile-devs

### CODE REVIEW

**Comment by elgatovital** [comment]
@biafra23 Where is the paragraph?

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -573,5 +573,5 @@
 
     <string name="whatsnew_title">What\'s new</string>
     <string name="whatsnew_item1_title">Meet the new Safe!</string>
-    <string name="whatsnew_item1_description">Gnosis Safe rebranded to Safe following a successful spin-off from the Gnosis DAO.\n\nNow and over the coming versions, you will see a new look and a better user experience for your ever secure Gnosis Safe.</string>
+    <string name="whatsnew_item1_description">&lt;b&gt;Gnosis Safe &lt;/b&gt; rebranded to &lt;b&gt;Safe&lt;/b&gt;. Following a successful spin-off vote from Gnosis in GIP-29, we are rebranding to Safe.\n\nNow and over the coming time, you will see a new look and a better visual experience to your ever secure Gnosis Safe that you love.</string>
```
Comment (elgatovital) [inline app/src/main/res/values/strings.xml:576]: I would rather work with Spannables and set ForegroundColorSpan for highlighted elements

**Review by elgatovital — CHANGES_REQUESTED** [review]

**Comment by elgatovital** [comment]
@biafra23 could you also replace `ic_app_24dp.xml` with the new logo in this pr?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (elgatovital) [reply app/src/main/res/values/strings.xml:576]: also highlighted elements are not bold, but have different color in design mocks

**Comment by biafra23** [comment]
> @biafra23 could you also replace ic_app_24dp.xml with the new logo in this pr?

I wanted to do that in a separate PR.

File: app/src/main/res/drawable/ic_app_24dp.xml [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+        android:fillColor="#ffffff"/>
+    <path
+        android:pathData="M289.19,320C289.19,303.17 302.81,289.53 319.62,289.53H543.39C560.2,289.53 573.82,275.89 573.82,259.06V202.7C573.82,185.87 560.2,172.23 543.39,172.23H306.65C289.84,172.23 276.22,185.87 276.22,202.7V246.13C276.22,262.96 262.6,276.6 245.79,276.6H202.48C185.67,276.6 172.05,290.24 172.05,307.07V402.16C172.05,418.99 185.73,431.95 202.54,431.95H258.82C275.63,431.95 289.25,418.31 289.25,401.48L289.19,320.01V320Z"
+        android:fillColor="#ffffff"/>
+    <path
+        android:pathData="M405.09,372.72H459.15C476.77,372.72 491.06,387.04 491.06,404.67V458.8C491.06,476.45 476.76,490.76 459.15,490.76H405.09C387.47,490.76 373.18,476.43 373.18,458.8V404.67C373.18,387.03 387.48,372.72 405.09,372.72Z"
+        android:fillColor="#ffffff"/>
```
[the remark this answers is not in the document]
Comment (biafra23) [reply app/src/main/res/drawable/ic_app_24dp.xml]: image

Looks good now

**Review by elgatovital — APPROVED** [review]

**Pull request merged**